### PR TITLE
Dev trunks le

### PR DESCRIPTION
**Pull request by OptimusPrime2001**

### CODE REVIEW

File: src/guard/role.guard.ts
Comment (benbeckman2001) [inline src/guard/role.guard.ts:1]: this code seem good

Comment (OptimusPrime2001) [reply src/guard/role.guard.ts:1]: ok

File: src/cats/cat.controller.ts [hunk 1]
```diff
@@ -5,23 +5,37 @@ import {
   Get,
   HttpStatus,
   Param,
+  ParseIntPipe,
```
Comment (benbeckman2001) [inline src/cats/cat.controller.ts:8]: aaaaa

**Review by benbeckman2001 — COMMENTED** [review]
aaa

**Pull request closed without merging**